contracts/tg4-engagement/Cargo.toml

```diff
@@ -21,20 +21,20 @@ library = []
 
 [dependencies]
 cosmwasm-std = "1.1.0"
-cw-controllers = "0.15.0"
-cw-storage-plus = "0.15.0"
-cw-utils = "0.15.0"
-cw2 = "0.15.0"
-tg-utils = { version = "0.15.2", path = "../../packages/utils" }
-tg-bindings = { version = "0.15.2", path = "../../packages/bindings" }
-tg4 = { path = "../../packages/tg4", version = "0.15.2" }
+cw-controllers = "0.16.0"
+cw-storage-plus = "0.16.0"
+cw-utils = "0.16.0"
+cw2 = "0.16.0"
```

Comment:
There are newer versions available.
https://github.com/CosmWasm/cw-plus/releases/tag/v1.0.0
https://github.com/CosmWasm/cw-storage-plus/releases/tag/v1.0.1
https://github.com/CosmWasm/cw-utils/releases/tag/v1.0.1

Reply:
Will give it a try.